script/is-it-pretty.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const prettier = require('prettier');
+const glob = require('glob');
+const fs = require('fs');
+const globPattern = 'app/{src, test}/**/*.{ts, tsx}'
+const prettierOptions = {
```

Comment:
Unfortunately no, the defaults don't like typescript or other things our linters like.